config/webpack/demo/webpack.config.dev.js

```diff
@@ -49,7 +49,7 @@ module.exports = {
         test: /\.js$/,
         // Use include specifically of our sources.
         // Do _not_ use an `exclude` here.
-        include: FILES.concat([`${DEMO}/js`]),
+        include: FILES.concat([path.normalize(`./${DEMO}/js`)]),
```

Comment:
Everything works fine if I drop the leading dot, however: 
```
include: FILES.concat([path.normalize(`/${DEMO}/js`)]),
```

Does that work for you?

Reply:
It works when I drop the slash as well. Is that better?